fault-tolerance/src/main/java/io/helidon/faulttolerance/Retry.java

```diff
@@ -640,6 +641,387 @@ public Builder config(Config config) {
+                return this;
+            }
+
+            /**
+             * Max Delay in Milliseconds.
+             *
+             * @param maxDelay Duration
+             * @return updated builder instance
+             */
+            @ConfiguredOption("180000")
+            public Builder maxDelay(Duration maxDelay) {
+                this.maxDelay = maxDelay;
+                return this;
+            }
+
+
+            /**
+             * Random part of the delay.
+             * A number between {@code [-jitter,+jitter]} is applied to delay each time
+             * delay is calculated.
+             *
+             * @param jitter jitter duration
+             * @return updated builder instance
+             */
+            @ConfiguredOption("50")
+            public Builder jitter(long jitter) {
+                this.randomJitter = jitter;
+                return this;
+            }
+
+            /**
+             * <p>
+             * Load all properties for this Retry Policy from configuration.
+             * </p>
+             * <table class="config">
+             * <caption>Configuration</caption>
+             * <tr>
+             *     <th>key</th>
+             *     <th>default value</th>
+             *     <th>description</th>
+             * </tr>
+             * <tr>
+             *     <td>calls</td>
+             *     <td>10</td>
+             *     <td>Number of calls</td>
+             * </tr>
+             * <tr>
+             *     <td>initial-delay</td>
+             *     <td>2</td>
+             *     <td>Initial delay</td>
+             * </tr>
+             * <tr>
+             *     <td>max-delay</td>
+             *     <td>3 minutes</td>
+             *     <td>Maximum delay</td>
+             * </tr>
+             * <tr>
+             *     <td>jitter</td>
+             *     <td>50 milliseconds</td>
+             *     <td>A number between {@code [-jitter,+jitter]} applied to delay</td>
+             * </tr>
+             * </table>
+             *
+             * @param config the config node to use
+             * @return updated builder instance
+             */
+            public Builder config(Config config) {
+                config.get("calls").asInt().ifPresent(this::calls);
+                config.get("initial-delay").as(Duration.class).ifPresent(this::initialDelay);
+                config.get("max-delay").as(Duration.class).ifPresent(this::maxDelay);
+                config.get("jitter").asLong().ifPresent(this::jitter);
+                return this;
+            }
+        }
+    }
+
+
+    /**
+     * A retry policy that increases the delay time following an exponential sequence.
+     * Allowed elements that are also annotated with {@code @Retry}.
+     * Expected sequence: initial delay, 2 * initial delay + jitter, 4 * initial delay + jitter,
+     * 8 * initial delay + jitter, etc. {@code maxDelayInMillis} is used to prevent endless waiting.
+     */
```

Comment:
This explanation of "expected sequence" does not mention `factor`. How is it used?

Reply:
fixed